### PR TITLE
CARRY: kueue manager needs RBACs to get,list,watch AppWrappers

### PR DESCRIPTION
**Pull request by dgrove-oss**

### CODE REVIEW

File: pkg/controller/jobframework/integrationmanager.go [hunk 1]
```diff
@@ -101,6 +101,8 @@ func (m *integrationManager) register(name string, cb IntegrationCallbacks) erro
 	return nil
 }
 
+//+kubebuilder:rbac:groups=workload.codeflare.dev,resources=appwrappers,verbs=get;list;watch
```
Comment (astefanutti) [inline pkg/controller/jobframework/integrationmanager.go:104]: We've refrained from patching Go sources. Could these extra RBAC be added "statically" in the `rhoai` overlay instead?

Comment (dgrove-oss) [reply pkg/controller/jobframework/integrationmanager.go:104]: I'll take a look.  I think it could be done that way.

**Review by astefanutti — APPROVED** [review]
Nice!

**Pull request merged**